cli/cmd/component-apply.go

```diff
@@ -26,10 +26,10 @@ import (
 )
 
 var componentApplyCmd = &cobra.Command{
-	Use: "apply",
-	Short: `Apply a component configuration. If not present it will install it.
-If ran with no arguments it will apply all components mentioned in the
-configuration.`,
+	Use:   "apply",
+	Short: "Deploy or update a Lokomotive component",
+	Long: `Idempotently deploy a Lokomotive component.
```

Comment:
I'd rather use a more clear description (i.e. I don't love the word "idempotently").
I think the short description already makes it clear, but if it's needed to be reenforced in the long one, maybe something like:
`Installs a component if it's not yet present, otherwise it updates it (applies the new confirguration).`

(The line below is fine)

Reply:
I've rephrased the long description of both `component apply` and `cluster apply` since they are similar in nature.